src/services/paras/ParasService.ts

```diff
@@ -429,6 +431,53 @@ export class ParasService extends AbstractService {
 		};
 	}
 
+	/**
+	 * Retrieve all the headers of the parachains connected to the relay chain given
```

Comment:
I would update this comment  as per my comments above. It is not all the parachain heads but those included/backed in the relevant relaychain block